templates/named.conf.erb

```diff
@@ -8,11 +8,6 @@ logging {
                 severity dynamic;
         };
 };
-zone "." IN {
-       type hint;
-       file "<%= @root_hint %>";
```

Comment:
To the best of my knowledge, no ,RedHat does not put the root zone hint in the rfc1912 zones file.  I will answer for sure once I manage to get my VMs working for testing puppet stuff on different OSes.